R/spatial.R

```diff
@@ -13,17 +13,17 @@ angle_between <- function(a, b, origin = vector3(0, 0, 0)) {
   # angle between is dot divided by length.
   a <- a - origin
   b <- b - origin
-  acos(dot(a, b) / (magnitude(a) * magnitude(b)))
+  acos(dot(a, b) / (magnitude(a, "L2") * magnitude(b, "L2")))
 }
 
-#' Cacluate distance between vectors
+#' Calculate distance between vectors
 #'
 #' Measure the distance between two points `from` and `to`
 #'
 #' @param from,to Endpoints of the distance to be calculated
```

Comment:
Add in an additional `@param` line describing the norms - the documentation style is Roxygen, for reference.

Reply:
Updated function documentation to have an additional `@param norm`

Reply:
Ope! One more thing to do - this is an R-packages thing, but I didn't remember to mention it. 
In order to generate the documentation files from the Roxygen comments, you need to run `devtools::document()` (and potentially `install.packages("devtools")` beforehand) - this is described more in https://r-pkgs.org/man.html#man-workflow if you need it.